java/source/avro.rst

```diff
@@ -0,0 +1,75 @@
+.. Licensed to the Apache Software Foundation (ASF) under one
+.. or more contributor license agreements.  See the NOTICE file
+.. distributed with this work for additional information
+.. regarding copyright ownership.  The ASF licenses this file
+.. to you under the Apache License, Version 2.0 (the
+.. "License"); you may not use this file except in compliance
+.. with the License.  You may obtain a copy of the License at
+
+..   http://www.apache.org/licenses/LICENSE-2.0
+
+.. Unless required by applicable law or agreed to in writing,
+.. software distributed under the License is distributed on an
+.. "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+.. _arrow-avro:
+
+===============================
+Avro to Arrow
+===============================
+
+Avro encoded data can be converted into Arrow format.
+
+.. contents::
+
+Avro to Arrow
+=============
+
+The example assumes that the avro data doesn't contain schema in the file
+to be converted into Arrow
```

Comment:
I honestly do not know what this means? The code sample is reading a schema it appears

Reply:
Ah, separate files.

```suggestion
The example assumes that the Avro schema is stored separately from the Avro data itself.
```

Reply:
Arrow data file can store schema along with data. This is in reference to that. This example shall fail in that case, hence mentioned explicitly.